src/server/logging/index.js

```diff
@@ -27,6 +27,6 @@ export async function setupLogging(server, config) {
   });
 }
 
-export function loggingMixin(kbnServer, server, config) {
-  return setupLogging(server, config);
+export async function loggingMixin(kbnServer, server, config) {
+  return await setupLogging(server, config);
```

Comment:
curious what was happening here, iirc this is already a promise?

Reply:
yeah exactly @jbudz

Reply:
I guess I was wondering why change it to async?  it should work same without the change.

Reply:
I guess I just changed it for coherence as I found the majority of the kbn_server mixins are async functions itself. But ofc I can revert the change here @jbudz 😃

Reply:
np fine as is, just didn't know if i was missing something